info_test.go

```diff
@@ -405,7 +405,7 @@ func TestProgInfoExtBTF(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	expectedLineInfoCount := 26
+	expectedLineInfoCount := 28
```

Comment:
That's the downside of overloading `loader.c`, I guess. To be honest, I do not understand where 26 is coming from, but it (kind of) makes sense that the value grew by 2 after two lines were added. @ti-mo

Reply:
Good question, that's quite straightforward to investigate:

```diff
		if src := ins.Source(); src != nil {
			lineInfoCount++
+			fmt.Println(src)
+			fmt.Println(ins)
		}
```

Results in:

```
...
AddReg dst: r0 src: r6
	return static_fn(arg) + global_fn(arg) + arg2;
LoadMapValue dst: r1, fd: 40 off: 0
	return static_fn(arg) + global_fn(arg) + arg2;
AddReg dst: r0 src: r1
	return static_fn(arg) + global_fn(arg) + arg2;
Exit

int __attribute__((noinline)) global_fn(uint32_t arg) {
...
	return static_fn(arg) + global_fn2(arg) + global_fn3(arg);
AddReg dst: r8 src: r0
	return static_fn(arg) + global_fn2(arg) + global_fn3(arg);
MovReg dst: r0 src: r8
```

One instruction for the variable access and one for accumulating into r0. (this is on `main`) Adding another variable to the `return` statement would result in 2 extra insns/lineinfo.